### PR TITLE
fix(docs): replace links to concerto from Hyperledger to Accord repo

### LINKED ISSUE #90: Concerto Links
**Describe the bug**
Documentation links to `concerto` still refer to Hyperledger's repositories

**To Reproduce**
The highlighted line here has an incorrect link:
Screen Shot 2019-09-18 at 16 25 31 PM

**Expected behavior**
Correct these links to https://github.com/accordproject/concerto

**Additional context**
There may be further broken links, search through the docs

### PR DESCRIPTION
**Pull request by daniloff200**
# Issue #90

### Changes
Replace Concerto links

closes #90

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -164,7 +164,7 @@ const Model = props => (
   <Block id="model">
     {[
       {
-        content: '<div class="typeset">The template model defines the structure of data that are used in the natural language and logic. Accord Project templates use the <a href="https://github.com/hyperledger/composer-concerto">Composer Concerto Modeling Language</a>.</div>',
+        content: '<div class="typeset">The template model defines the structure of data that are used in the natural language and logic. Accord Project templates use the <a href="https://github.com/accordproject/concerto">Composer Concerto Modeling Language</a>.</div>',
```
Comment (mttrbrts) [inline website/pages/en/index.js]: Can you remove the reference to Concerto on this line, please?

File: website/versioned_docs/version-0.10.0/accordproject-specification.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ The first step in converting the late delivery and penalty clause to use Accord
...
+They are captured formally using the [Concerto modeling language](https://github.com/accordproject/concerto) (CML). CML is a lightweight schema language that defines namespaces, types, and relationships between types. It includes first-class support for modeling participants (individuals or companies), assets, transactions, enumerations, concepts, and events, and includes the typical features of an Object Oriented modeling language, including inheritance, meta-annotations (decorators), and field specific validators. CML also defines a serialization of instances to JSON and a validator for instances, making it easy to integrate with a wide variety of JSON-capable external systems.
```
Comment (mttrbrts) [inline website/versioned_docs/version-0.10.0/accordproject-specification.md]: This is a static versioned file and shouldn't be edited, please remove this change.

When we make a new release of the documentation, Docusarus creates these `versions_docs` automatically as a snapshot of the documentation for historical reference.

File: website/versioned_docs/version-0.10.0/ergo-intro-examples.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ This declares that this module belongs to the `org.accordproject.helloworld` nam
 
 It also declares that the contract `HelloWorld` is parameterized over a given `TemplateModel`.
 
-The `TemplateModel` as well as the `Request` and `Response` are types which are specified using [Composer Concerto modeling language](https://github.com/hyperledger/composer-concerto).
+The `TemplateModel` as well as the `Request` and `Response` are types which are specified using [Composer Concerto modeling language](https://github.com/accordproject/concerto).
```
Comment (mttrbrts) [inline website/versioned_docs/version-0.10.0/ergo-intro-examples.md]: This is a static versioned file and shouldn't be edited, please remove this change.

When we make a new release of the documentation, Docusarus creates these `versions_docs` automatically as a snapshot of the documentation for historical reference.

File: website/versioned_docs/version-0.10.0/ergo.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ To achieve those goals the design of Ergo is based on the following
 principles:
 
 - Ergo contracts have a class-like structure with clauses akin to methods
-- Ergo can handle types (concepts, transations, etc) defined with the [Composer Concerto Modeling Language](https://github.com/hyperledger/composer-concerto) (so called CML models), as mandated by the Accord Project Template Specification
+- Ergo can handle types (concepts, transations, etc) defined with the [Composer Concerto Modeling Language](https://github.com/accordproject/concerto) (so called CML models), as mandated by the Accord Project Template Specification
```
Comment (mttrbrts) [inline website/versioned_docs/version-0.10.0/ergo.md]: This is a static versioned file and shouldn't be edited, please remove this change.

When we make a new release of the documentation, Docusarus creates these `versions_docs` automatically as a snapshot of the documentation for historical reference.

File: website/versioned_docs/version-0.10.1/accordproject-specification.md [hunk 5]
```diff
@@ -97,7 +97,7 @@ The first step in converting the late delivery and penalty clause to use Accord
...
+They are captured formally using the [Concerto modeling language](https://github.com/accordproject/concerto) (CML). CML is a lightweight schema language that defines namespaces, types, and relationships between types. It includes first-class support for modeling participants (individuals or companies), assets, transactions, enumerations, concepts, and events, and includes the typical features of an Object Oriented modeling language, including inheritance, meta-annotations (decorators), and field specific validators. CML also defines a serialization of instances to JSON and a validator for instances, making it easy to integrate with a wide variety of JSON-capable external systems.
```
Comment (mttrbrts) [inline website/versioned_docs/version-0.10.1/accordproject-specification.md]: This is a static versioned file and shouldn't be edited, please remove this change.

When we make a new release of the documentation, Docusarus creates these `versions_docs` automatically as a snapshot of the documentation for historical reference.

File: website/pages/en/index.js (hunk 1 above)
Comment (daniloff200) [reply website/pages/en/index.js]: hello !
do you mean complete removal of `<a></a>`, or, just some hyperlink fix?

Comment (daniloff200) [reply website/pages/en/index.js]: @mttrbrts  ping :)

Comment (irmerk) [reply website/pages/en/index.js]: I believe @mttrbrts is saying to remove the second sentence "_Accord Project templates use the ..._"

Comment (daniloff200) [reply website/pages/en/index.js]: @irmerk  well, sounds reasonable

gonna do that then

File: website/pages/en/index.js [hunk 6]
```diff
@@ -164,7 +164,7 @@ const Model = props => (
   <Block id="model">
     {[
       {
-        content: '<div class="typeset">The template model defines the structure of data that are used in the natural language and logic. Accord Project templates use the <a href="https://github.com/hyperledger/composer-concerto">Composer Concerto Modeling Language</a>.</div>',
+          content: '<div class="typeset">The template model defines the structure of data that are used in the natural language and logic. Accord Project templates use the <a href="https://github.com/hyperledger/composer-concerto">Composer Concerto Modeling Language</a>.</div>',
```
Comment (irmerk) [inline website/pages/en/index.js]: I think this link still needs to be changed.

Comment (daniloff200) [reply website/pages/en/index.js]: well
If I remove this sentence, than, I think not :D

Comment (irmerk) [reply website/pages/en/index.js]: AH! Yes, I think it's still early for me, I was a bit confused.

Comment (irmerk) [reply website/pages/en/index.js]: @mttrbrts I think just still waiting on confirmation on the removal of the second sentence.

File: docs/example-eatapple.md [hunk 7]
```diff
@@ -22,7 +22,7 @@ variables such as `employee`, `company` and `tax`.
 
 The second component of a smart legal template is the model, which is
 expressed using the [Composer Concerto modeling
```
Comment (jeromesimeon) [inline docs/example-eatapple.md]: The name for it should also be changed. Please remove`Composer` everywhere referring to the Concerto modelling language.

File: website/pages/en/index.js (hunk 6 above)
Comment (jeromesimeon) [reply website/pages/en/index.js]: I'm confused by this thread, I don't see a sentence removed and it still uses the old link and name for Concerto.

Comment (daniloff200) [reply website/pages/en/index.js]: @jeromesimeon the main question was : should it be removed completely, or not? I revert all changes back before getting the final requirements, about the name - yes, that will be done

**Comment by daniloff200** [comment]
Well, okay :)
So small tasks, but, so many comments :) 

I removed sentence, mentioned by @mttrbrts & @irmerk 
Also, removed  Composer everywhere referring to the Concerto modelling language.

File: docs/basic-templates.md [hunk 8]
```diff
@@ -91,7 +91,7 @@ transaction MyResponse extends Response {
 }
 ```
 
-The `TemplateModel` as well as the `Request` and `Response` are types which are specified using the [Composer Concerto modeling language](https://github.com/hyperledger/composer-concerto).
+The `TemplateModel` as well as the `Request` and `Response` are types which are specified using the [Concerto modeling language](https://github.com/hyperledger/composer-concerto).
```
Comment (irmerk) [inline docs/basic-templates.md]: Missed one link update!

Comment (daniloff200) [reply docs/basic-templates.md]: OOOOOOOOOOOOOOOOHHHHHHH

Comment (daniloff200) [reply docs/basic-templates.md]: You hadn't seen this :rofl:

**Pull request merged**